src/DotNet.ReproducibleBuilds/DotNet.ReproducibleBuilds.nuspec

```diff
@@ -21,8 +21,10 @@
   </metadata>
   <files>
     <file src="DotNet.ReproducibleBuilds.props" target="build\DotNet.ReproducibleBuilds.props" />
-    <file src="DotNet.ReproducibleBuilds.props" target="buildMultiTargeting\DotNet.ReproducibleBuilds.props" />
+    <file src="DotNet.ReproducibleBuilds.shim.props" target="buildMultiTargeting\DotNet.ReproducibleBuilds.props" />
+    <file src="DotNet.ReproducibleBuilds.shim.props" target="buildTransitive\DotNet.ReproducibleBuilds.props" />
     <file src="DotNet.ReproducibleBuilds.targets" target="build\DotNet.ReproducibleBuilds.targets" />
```

Comment:
This and the first item appear to be missing given the file renaming.

Reply:
I'm not sure what you want here. The new structure is:

- `build/<PackageName>.props/target` contain the full xml
- `buildMultiTargeting/<PackageName>.props/target` contain the shim
- `buildTransitive/<PackageName>.props/target` contain the shim

Do you prefer another way to store files in the nupkg?

<img width="263" alt="image" src="https://user-images.githubusercontent.com/509220/181524598-a1352da1-ff8e-4ac8-8bc4-e93ef135e1a9.png">

Reply:
Oh, sorry, you're right, I got confused when reviewing the PR in the mobile app.